ci/compute-build-args.sh

```diff
@@ -3,7 +3,6 @@ set -euo pipefail
 
 MANYLINUX_VER="manylinux_2_17"
```

Comment:
@bdice do we want to update this to `manylinux_2_28`?

Reply:
Yes, but an even better solution would be to error if we’re using an image not listed below (Ubuntu 20.04 or Rocky 8). In my understanding, we must know and specify the correct glibc version for the OS here.